docs/environment-integrations/swarm-rl.md

```diff
@@ -0,0 +1,64 @@
+# Quad-Swarm-RL Integrations
+
+### Installation
+
+Clone https://github.com/Zhehui-Huang/quad-swarm-rl into your home directory
```

Comment:
Maybe mention a specific branch/commit you tested this on, because that repo might keep changing and the integration will break.